waku/v2/node/storage/message/waku_message_store.nim

```diff
@@ -114,7 +115,15 @@ method getAll*(db: WakuMessageStore, onData: message_store.DataProc): MessageSto
            WakuMessage(contentTopic: contentTopic, payload: payload , version: uint32(version), timestamp: senderTimestamp.float64), 
                        pubsubTopic)
 
-  let res = db.database.query("SELECT receiverTimestamp, contentTopic, payload, pubsubTopic, version, senderTimestamp FROM " & TABLE_TITLE & " ORDER BY receiverTimestamp ASC", msg)
+  var selectQuery = "SELECT receiverTimestamp, contentTopic, payload, pubsubTopic, version, senderTimestamp " &
+                    "FROM " & TABLE_TITLE & " " & 
+                    "ORDER BY receiverTimestamp ASC"
```

Comment:
Might need index for this if we don't have this already

Are we logging before and after this operation so we can debug long access time in cluster logs?

Even if debug logs, if we only log info that might not be visible

Reply:
> Might need index for this if we don't have this already

There is a primary ID per record taking the message index, if that is what you're referring to? We just don't access these when reading.

> Are we logging before and after this operation so we can debug long access time in cluster logs? Even if debug logs, if we only log info that might not be visible

Good point! I've added some before and after logs now.

Reply:
Nevermind actually, I forgot we only do this select at startup time :man_facepalming: